src/lib/processOpenAPI/getCodeSamples.ts

```
@@ -0,0 +1,56 @@
+      if (!operation) {
+        continue
+      }
+
+      const authorizations = operation.securityUi?.[0]?.schemes || []
+
+      const request = buildRequest({
+        path,
+        method: verb,
+        baseUrl,
+        parameters: operation.parameters || [],
+        authorizations: Object.entries(authorizations).map(([name, value]) => {
```

ESLint

[error] 'value' is defined but never used. Allowed unused args must match /^_/u.

(unused-imports/no-unused-vars)